frontend/src/utils/api.js

```diff
@@ -55,30 +55,58 @@ function getEventAttendees(id: string) {
   return fetch(`${API_URL}/events/${id}/attendees?key=${getKey()}`).then(res => res.json())
 }
 
-function addInterviewSchedule(file: File) {
-  var reader = new FileReader()
-  var scheduleString = ''
-  reader.onload = function(e) {
-    scheduleString = reader.result
-    fetch(`${API_URL}/schedule/upload/?key=${getKey()}`, {
-      method: 'POST',
-      headers: {
-        'content-type': 'application/json'
-      },
-      body: JSON.stringify({ schedule: scheduleString })
-    })
-      .then(res => res.json())
-      .then(success => console.log(success))
-      .catch(error => console.log(error))
-  }
-
-  reader.readAsText(file)
+async function addInterviewerSchedules(file: File) {
```

Comment:
you can probably merge this function with the one below; something like
```
addSchedules(file: File, scheduleType) 
```
and based on the the string passed in fetch to 
```
${API_URL}/schedule/upload${scheduleType}/?key=${getKey()}
```

Reply:
Yeah I definitely agree these can be combined.